stan/math/prim/scal/meta/ad_promotable.hpp

```diff
@@ -107,26 +108,42 @@ struct ad_promotable<unsigned long, double> {  // NOLINT(runtime/int)
 };
 
 /**
- * An unsigned int may be promoted to a double.
+ * A long long may be promoted to a double.
```

Comment:
@seantalts: `long long` and `unsigned long long` are the two new specializations. (If you remove these two and try to run on Windows, it'll fail to compile.)